console/console-init/ui/src/modules/iot-device/components/DeviceListToolbar/DeviceListToolbar.test.tsx

```diff
@@ -74,7 +74,7 @@ describe("<DeviceListToolbar />", () => {
     getByText("Upload a JSON file");
   });
 
-  it("should render appropriate options for BulkSelect", () => {
+  xit("should render appropriate options for BulkSelect", () => {
```

Comment:
is there any reason for skip this test. if yes, then comment the reason so that we can fix later.